fairlearn/metrics/_metric_frame.py

```diff
@@ -405,6 +565,29 @@ def overall(self) -> Union[Any, pd.Series, pd.DataFrame]:
         else:
             return self._overall
 
+    @property
+    def overall_ci(
+        self,
+    ) -> Optional[List[Tuple[float, Union[pd.Series, pd.DataFrame]]]]:
+        """Return estimates of overall confidence intervals calculated via bootstrap."""
+
+        if self._overall_ci:
+            if self._user_supplied_callable:
+                if self.control_levels:
+                    return [
+                        (quantile, output.iloc[:, 0])
+                        for quantile, output in self._overall_ci
+                    ]
+                else:
+                    return [
+                        (quantile, output.iloc[0])
+                        for quantile, output in self._overall_ci
+                    ]
+            else:
+                return self._overall_ci
+
+        return None
```

Comment:
To me `None` makes sense, because that it throws an error if you try to iterate over `_overall_ci` which might be useful for debugging purposes. But I don't know what the best practices are :)